pkg/webui/console/views/gateway-add/index.js

```diff
@@ -119,6 +119,7 @@ export default class GatewayAdd extends React.Component {
               onSubmit={this.handleSubmit}
               initialValues={initialValues}
               update={false}
+              formRef={React.createRef()}
```

Comment:
What is the purpose of passing ref as a prop without actually using it in `<GatewayAdd />`?